### PR TITLE
polish(gry): polish dqn config

### PR DESCRIPTION
**Pull request by ruoyuGao**
## Description
find all the possible dqn parameters and add them into dqn default config and comment. 
## Check List

- [x] merge the latest version source branch/repo, and resolve all the conflicts
- [x] pass style check
- [x] pass all the tests

### CODE REVIEW

File: ding/policy/dqn.py [hunk 1]
```diff
@@ -38,30 +38,41 @@ class DQNPolicy(Policy):
...
+        8  | ``model.dueling``  bool    True            | dueling head architecture
+        9  | ``model.encoder``  list(int)[32, 64,       | Sequence of ``hidden_size`` of         | default kernel_size
+           | ``_hidden_size_list``        64, 128]      | subsequent conv layers and the         | is [8, 4, 3]
+                                                        | final dense layer.                     | default stride is
+                                                                                                 | [4, 2 ,1]
+        10 | ``learn.update``   int      3              | How many updates(iterations) to train  | This args can be vary
            | ``per_collect``                            | after collector's one collection. Only | from envs. Bigger val
                                                         | valid in serial training               | means more off-policy
-        9  | ``learn.multi``    bool     False          | whether to use multi gpu during
+        11 | ``learn.multi``    bool     False          | whether to use multi gpu during
```
Comment (PaParaZz1) [inline ding/policy/dqn.py:49]: during?

File: ding/policy/dqn.py [hunk 2]
```diff
@@ -98,15 +115,18 @@ class DQNPolicy(Policy):
...
             ignore_done=False,
         ),
         # collect_mode config
         collect=dict(
             # (int) Only one of [n_sample, n_episode] shoule be set
-            # n_sample=8,
+            n_sample=8,
             # (int) Cut trajectories into pieces with length "unroll_len".
             unroll_len=1,
         ),
```
Comment (PaParaZz1) [inline ding/policy/dqn.py:132]: add comment for replay_buffer_size

File: ding/policy/dqn.py [hunk 3]
```diff
@@ -84,6 +95,12 @@ class DQNPolicy(Policy):
         discount_factor=0.97,
         # (int) The number of step for calculating target q_value
         nstep=1,
+        model=dict(
+            #(list(int)) Sequence of ``hidden_size`` of subsequent conv layers and the final dense layer.
+            encoder_hidden_size_list=[128, 128, 64],
+            # (bool) whether enable dueling head
```
Comment (PaParaZz1) [inline ding/policy/dqn.py:101]: Each comment line should start from upper case and end with a period

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: ding/policy/dqn.py (hunk 2 above)
Comment (ruoyuGao) [reply ding/policy/dqn.py:132]: added

File: ding/policy/dqn.py (hunk 3 above)
Comment (ruoyuGao) [reply ding/policy/dqn.py:101]: fixed all

File: ding/policy/dqn.py (hunk 1 above)
Comment (ruoyuGao) [reply ding/policy/dqn.py:49]: removed

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**